### DISCUSSION TITLE
Behavior Analytics for Pixi.js Applications - Recommendations Needed

### DISCUSSION THREAD

**Discussion by yordan-kanchelov** [pixijs/pixijs#9532] [General]
Hey guys,

I've been trying to find a good behavior analytics solution for our game application.  We want to gain a deeper understanding of how users interact with our application, similar to what Hotjar offers for HTML-based sites.

However, as Hotjar and many other traditional analytics tools don't support tracking on canvas elements, we're facing a bit of a roadblock. Has anyone in this community found or used any services, frameworks, or custom solutions for behavior analytics with Pixi.js applications? We're particularly interested in features like heatmaps, session recordings, and user feedback.

**Comment by shivavelingker** [comment]
Did you have any discoveries here? Facing a similar issue

**Comment by koteelok** [comment]
Developing a pixi.js-native analytics solution could be a good way for the pixi.js team to finance the framework’s development.

And it kind of would suit pixi since it's a "creation engine"
